tests/scripts/generate_with_quotes.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Handling quotes in code generation
```

Comment:
I would instead define this whole feature file in a docstring in test_generate_with_quotes and write it to a file in the testdir programmatically, just like we do in https://github.com/pytest-dev/pytest-bdd/blob/f4dce04053036f10350ce682396c34c6d8cc4dec/tests/feature/test_no_sctrict_gherkin.py#L4